Проблемы с запуском

Выполнил первые 3 шага из доки:
* git clone
* npm i
* npm start 

Вот такой лог выдало:
```bash
~/Work/new-project master= $ npm start

> csssr-project-template@0.25.0 start /Users/a.malko/Work/new-project
> gulp --debug

[13:09:12] Failed to load external module babel-register
[13:09:12] Requiring external module babel-core/register
module.js:327
    throw err;
    ^

Error: Cannot find module 'stylus'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/a.malko/Work/new-project/node_modules/stylus-import-if-exist/dist/index.js:7:15)
    at Module._compile (module.js:409:26)
    at Module._extensions..js (module.js:416:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/a.malko/Work/new-project/node_modules/babel-core/node_modules/babel-register/lib/node.js:134:7)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (styles.js:7:1)
    at Module._compile (module.js:409:26)
    at loader (/Users/a.malko/Work/new-project/node_modules/babel-core/node_modules/babel-register/lib/node.js:126:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/a.malko/Work/new-project/node_modules/babel-core/node_modules/babel-register/lib/node.js:136:7)

npm ERR! Darwin 15.3.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v4.3.1
npm ERR! npm  v2.14.12
npm ERR! code ELIFECYCLE
npm ERR! csssr-project-template@0.25.0 start: `gulp --debug`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the csssr-project-template@0.25.0 start script 'gulp --debug'.
npm ERR! This is most likely a problem with the csssr-project-template package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp --debug
npm ERR! You can get their info via:
npm ERR!     npm owner ls csssr-project-template
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/a.malko/Work/new-project/npm-debug.log
```

OS X 10.11.3 Node 4.3.1, npm 2.14.12

--- a/package.json
+++ b/package.json
@@ -109,6 +109,7 @@
     "spritesmith-stylus-retina-template": "^1.0.0",
     "stylint": "^1.3.6",
     "stylint-stylish": "^1.2.0",
+    "stylus": "^0.54.5",
     "stylus-import-if-exist": "^1.1.0",
     "stylus-svg-size-template": "^1.0.0",
     "svg4everybody": "^2.0.0",

